easybuild/easyblocks/g/gromacs.py

```diff
@@ -63,6 +63,8 @@ def extra_options():
         extra_vars.update({
             'double_precision': [None, "Build with double precision enabled (-DGMX_DOUBLE=ON), " +
                                  "default is to build double precision unless CUDA is enabled", CUSTOM],
+            'single_precision': [None, "Build with single precision enabled (-DGMX_DOUBLE=OFF), " +
```

Comment:
Actually, i changed my mind. This shows up in the documentation and i think it's good to show the default value.
```suggestion
            'single_precision': [True, "Build with single precision enabled (-DGMX_DOUBLE=OFF), " +
```

Reply:
done